apps/web/components/v2/apps/CalendarListContainer.tsx

```diff
@@ -153,7 +153,7 @@ function ConnectedCalendarsList(props: Props) {
           return null;
         }
         return (
-          <List className="flex flex-col gap-6">
+          <List className="flex flex-col gap-6" noBorderTreatment>
```

Comment:
Had to make the list without borders as they were messing up each calendar integration list item, specifically the warning itself.